recipes/dtool_azure/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dtool_azure" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/dtool-azure/dtool-azure-{{ version }}.tar.gz
+  sha256: e78ce652565689307aee40a7acf0c1d8a4209757588002afbcbc1e61b3dedb3a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - azure-storage-blob ==2.1.0
+    - azure-storage-common ==2.1.0
+    - dtoolcore >=3.17
+    - python >=3.6
+
+test:
+  imports:
+    - dtool_azure
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jic-dtool/dtool-azure
+  summary: Add Azure dataset support to dtool
+  license: MIT
+  license_file: LICENSE.rst
```

Comment:
Hi, can you please add [these fields](https://github.com/jan-janssen/staged-recipes/blob/8308324b4627e26748cdd80ab5129791e6d755ed/recipes/example/meta.yaml#L76-L82) if possible? Thanks!

Reply:
Done